packages/example-completeness/test/src/AttributeNormalizer.spec.js

```diff
@@ -1,16 +1,13 @@
-import AttributeNormalizer from '../../AttributeNormalizer';
-
+const AttributeNormalizer = require('../../src/AttributeNormalizer');
 const completeProduct = require('../resources/completeProduct.json');
 
-import { expect } from 'chai';
-
 describe('AttributeNormalizer', () => {
   it('should identify required locales based on the given product', () => {
     const localizations = AttributeNormalizer.getLocalizations(
       completeProduct,
       '$.masterData.staged.masterVariant.attributes.*'
     );
-    expect(localizations).to.have.lengthOf(2);
+    expect(localizations).toHaveLength(2);
```

Comment:
Just the matchers changed. Apart from that Jest is way more pleasant to work with. We get code coverage on CI and GitHub integration. As much as a `yarn test:watch` which spins up the super interactive Jest environment. For more see [here](https://github.com/commercetools/commercetools-serverless-examples/blob/master/package.json#L16).